config/tech-docs.yml

```diff
@@ -31,3 +31,10 @@ github_repo: alphagov/gds-way
 # Ownership for page reviews
 default_owner_slack: '#gds-way'
 owner_slack_workspace: gds
```

Comment:
What about adding `collapsible_nav: true`. To create a top level collapsible nav as per https://tdt-documentation.london.cloudapps.digital

https://github.com/alphagov/tdt-documentation/blob/master/config/tech-docs.yml#L28

Reply:
I tried this and it looks pretty good on the homepage but doesn't work on the lower level ages as all the content is by default hidden under the collapsed nav

<img width="981" alt="Screenshot 2020-02-19 at 08 47 58" src="https://user-images.githubusercontent.com/31240444/74817844-82043380-52f5-11ea-8672-5e5fd047e37d.png">

Reply:
Good point, I didn't check lower level pages.